ReactWindows/ReactNative.Shared/UIManager/ThemedReactContext.cs

```diff
@@ -1,12 +1,13 @@
-﻿using ReactNative.Bridge;
+﻿using System;
+using ReactNative.Bridge;
 
 namespace ReactNative.UIManager
 {
     /// <summary>
     /// A wrapper <see cref="ReactContext"/> that delegates lifecycle events to
     /// the original instance of <see cref="ReactContext"/>.
     /// </summary>
-    public class ThemedReactContext : ReactContext
```

Comment:
Instead of inheriting from ReactContext this class will now decorate the one and only ReactContext... The previous implementation results in two instances of ReactContext that share a single instance of the ReactInstance.  If the true intention of the is class is to provide a way to pass additional context to view related methods while providing access to the ReactContext then using the decorator pattern makes more sense...